index.html

```diff
@@ -191,10 +191,31 @@ <h3 id="About-me">About me</h3>
                 </div>
             </section>
         </div>
-
-
-        
-     </main>
+        <div class="footer">
+            <section id="contact-form" class="work-section">
+              <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+              <form action="#">
+                <input type="text" class="font-style name-email" name="fullname" id="fullname" placeholder="Full name">
```

Comment:
- [ ] A user cannot submit the form if one or more of the fields is empty. Please add the `required` attribute to every input field so a user can not submit the form if one or more of the fields is empty.